src/backend/catalog/ag_catalog.c

```diff
@@ -178,24 +177,19 @@ static void object_access(ObjectAccessType access, Oid class_id, Oid object_id,
      * The above applies to DROP TABLE command too.
      */
 
+    if (drop_arg->dropflags & PERFORM_DELETION_INTERNAL)
```

Comment:
Does this change break for the case `class_id == RelationRelationId`?

Reply:
No. It does not break.

Reply:
Could you elaborate more? In this case, the function calls `delete_label` first before returning. But with your change, it returns without calling that function. Does it have side effect?